devices.js

```diff
@@ -6379,6 +6402,131 @@ const devices = [
         fromZigbee: [fz.on_off],
         toZigbee: [tz.on_off],
     },
+
+    // Legrand
+    {
+        zigbeeModel: [' Shutter switch with neutral\u0000\u0000\u0000'],
+        model: '067776',
+        vendor: 'Legrand',
+        description: 'Netatmo wired shutter switch',
+        // the physical LED will be green when permit join is true, off otherwise and red when not linked
+        supports: 'open, close, stop, position, tilt',
+        fromZigbee: [
+            // Devices can send an identify message when the configuration button is pressed
+            // (behind the physical buttons)
+            // Used on the official gateway to send to every devices an identify command (green)
+            fz.identify,
+            fz.ignore_basic_report,
+            // support binary report on moving state (supposed)
+            fz.legrand_binary_input_moving,
+            fz.cover_position_tilt,
+        ],
+        toZigbee: [
+            tz.cover_state, // to verify
```

Comment:
Has this been verified?

Reply:
From what I sniffed with the official gateway that's what is required (identical) but couldn't tests (in a few days hopefully) on a real shutter.